lib/runtime/GlobalRuntimeModule.js

```diff
@@ -17,9 +17,9 @@ class GlobalRuntimeModule extends RuntimeModule {
 	 * @returns {string} runtime code
 	 */
 	generate() {
+		const { runtimeTemplate } = this.compilation;
 		return Template.asString([
-			`${RuntimeGlobals.global} = (function() {`,
-			Template.indent([
+			`${RuntimeGlobals.global} = ${runtimeTemplate.iife("", [
```

Comment:
revert this change. It must always be a `function()` because `this` is used in body.